helm/daq-config-server/values.yaml

```diff
@@ -48,9 +48,23 @@ readinessProbe:
     path: /healthz
     port: http
 
-volumes: []
+# Send logs to the dodal graylog stream for now until a more suitable stream exists
+logging_config:
+  graylog:
+    enabled: true
+    url: http://graylog-log-target.diamond.ac.uk:12232/
+    level: "INFO"
+
+volumes:
+  - name: config-volume
+    configMap:
+      name: {{ include "daq-config-server.fullname" . }}-config
```

Comment:
Is Helm happy with this when you `helm template`? I think for blueapi I had to add this explicitly to the pod because it didn't template this first